recipes/pyzbar/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyzbar" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: "https://github.com/NaturalHistoryMuseum/pyzbar"
+  git_rev: "v{{ version }}"
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - libtool
+
+test:
+  requires:
+    - pip
+    - pytest
+    - numpy
+    - pillow>=3.2.0
```

Comment:
Again I think if these are here they might need to be run instead.

Reply:
Removed